org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsControl.cs

```diff
@@ -915,6 +915,56 @@ private void TransformTarget()
                         //     transformUpdated = elasticsManager.ApplyTargetTransform(constraintTranslate, TransformFlags.Move);
                         // }
 
+                        if (!transformUpdated.IsMaskSet(TransformFlags.Move))
+                        {
+                            Target.transform.position = smoothingActive ?
+                                Smoothing.SmoothTo(Target.transform.position, constraintTranslate.Position, translateLerpTime, Time.deltaTime) :
+                                constraintTranslate.Position;
+                        }
+                    }
+                    else if (currentHandle.HandleType == HandleType.Translation2D)
+                    {
+                        Vector3 translateVectorOnPlane = Vector3.ProjectOnPlane(currentGrabPoint - initialGrabPoint, currentHandle.transform.forward);
+
+                        var goal = initialTransformOnGrabStart.Position + translateVectorOnPlane;
+                        MixedRealityTransform constraintTranslate = MixedRealityTransform.NewTranslate(goal);
+                        if (EnableConstraints && constraintsManager != null)
+                        {
+                            constraintsManager.ApplyTranslationConstraints(ref constraintTranslate, true, currentHandle.IsGrabSelected);
+                        }
+
+                        // TODO: Elastics integration (soon!)
+
+                        // if (elasticsManager != null)
+                        // {
+                        //     transformUpdated = elasticsManager.ApplyTargetTransform(constraintTranslate, TransformFlags.Move);
+                        // }
```

Comment:
Thank you for your contribution, this change looks great. We are trying to keep the codebase as clean as possible so we encourage contributors to avoid leaving commented code out of main branch. Could you please remove the commented code?